egui-wgpu/CHANGELOG.md

```diff
@@ -3,7 +3,8 @@ All notable changes to the `egui-wgpu` integration will be noted in this file.
 
 
 ## Unreleased
-Enables deferred render + surface state initialization for Android ([#1634](https://github.com/emilk/egui/pull/1634))
+* Enables deferred render + surface state initialization for Android ([#1634](https://github.com/emilk/egui/pull/1634)).
```

Comment:
I didn't plan on changing this but it was all on a single line without the formatting change.